src/data/v2/stream.rs

```diff
@@ -401,6 +437,9 @@ impl<const N: usize> From<[&'static str; N]> for Normalized {
 pub struct MarketData {
   /// The aggregate bars to subscribe to.
   pub bars: Normalized,
+
+  /// The aggregate quotes to subscribe to.
```

Comment:
What is being aggregated with quotes? Aren't they "raw"?

Reply:
fixed